Sources/TuistKit/Services/TestService.swift

```diff
@@ -204,8 +207,13 @@ final class TestService {
             throw TestServiceError.schemeWithoutTestableTargets(scheme: scheme.name)
         }
 
+        guard let platform = buildableTarget.target.exclusivePlatform else {
```

Comment:
Seeing the same changes in two places makes me think we should eventually extract the common functionality between `BuildService` and `TestService` into some shared piece of code (not in this PR)

Reply:
I was thinking the same but not sure what the appropriate levels of sharing were between services.  This could also potentially work towards